blackbox/docs/release_notes/1.1.3.txt

```diff
@@ -0,0 +1,73 @@
+.. _version_1.1.3:
+
+=============
+Version 1.1.3
+=============
+
+Released on 2017/05/09.
+
+.. NOTE::
+
+   If you are upgrading a cluster, you must be running CrateDB
+   :ref:`version_1.0.2` or higher before you upgrade to 1.1.3.
+
+   If you want to perform a :ref:`cluster_upgrade`, your current CrateDB
+   version number must be :ref:`version_1.1.2`. Any upgrade from a version
```

Comment:
Is this true? I was under the impression that for patch releases a cluster upgrade would be available from the first version in the feature version (1.1.1 in this case).